### PR TITLE
MAGEDOC-3058: Migration guide infographics

### PR DESCRIPTION
**Pull request by bdenham**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will add infographics to the "How migration works" topic in the Migration guide according to Story #MAGEDOC-3058.

whatsnew
The [How migration works](https://devdocs.magento.com/guides/v2.2/migration/migration-overview-how.html) topic was updated to clarify the migration process.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=2830) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=2830) before we can accept your contribution.<br/><hr/>**Bruce Denham** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.1/migration/migration-overview-how.md [hunk 1]
```diff
@@ -4,41 +4,60 @@ title: How migration works
...
 
-This guide provides information about migration using the Data Migration Tool. This tool transfers and adapts data from Magento 1 to Magento 2 stores by splitting the migration into three phases (or modes):
+The Data Migration Tool splits the migration process into three phases or *modes* in order to transfer and adapt data from Magento 1.x to Magento 2.x. The three modes are listed here and must be run in this order:
 
-*	Configuration settings
-*	Data
-*	Delta (changes since the last run)
+1. **Settings Mode**: migrates the system configuration and website-related settings.
+2. **Data Mode**: migrates database assets in bulk.
+3. **Delta Mode**: migrates incremental changes (changes since the last run), such as orders and inventory.
```
Comment (victor-v-rad) [inline guides/v2.1/migration/migration-overview-how.md]: >such as orders and inventory

such as new customers and orders.

File: guides/v2.1/migration/migration-overview-how.md [hunk 2]
```diff
@@ -4,41 +4,60 @@ title: How migration works
...
+The Data Migration Tool uses a list of *steps* within each mode to migrate a particular type of data. For example, in the Settings mode, there are two steps used to migrate the all the settings data: the Stores step and the Settings step. Details about the the specific data that is migrated in each of these steps (as well as for steps in the other modes), can be found in the [Data Migration Tool Technical Specification].
 
-This section previews your migration experience by providing a high-level overview of the Data Migration Tool.
+![Migration Overview]
 
-## Conceptual overview
+### Stages
+Within each step are three *stages* that are always executed in this order to ensure the data gets properly migrated: 
+1. **Integration Check**: Compares the table field names, types, 
and other info to verify compatibility between Magento 1 and 2 data structures.
```
Comment (victor-v-rad) [inline guides/v2.1/migration/migration-overview-how.md]: >Integration Check

Integrity Check

**Comment by bdenham** [comment]
Thanks, @victor-v-rad! BTW, that last section "Migration flow diagram" seems really disconnected from the other content in this topic. Is there a different place in the guide that you think we could move it to?

File: guides/v2.1/migration/migration-overview-how.md [hunk 3]
```diff
@@ -4,41 +4,64 @@ title: How migration works
 redirect_from: /guides/v1.0/migration/migration-overview-how.html
```
Comment (keharper) [inline guides/v2.1/migration/migration-overview-how.md]: It's OK to delete redirects that point to the `v1.0` directory. It's an artifact from the pre 2.0 days (2015) and Google won't be serving up those very stale links.

Comment (bdenham) [reply guides/v2.1/migration/migration-overview-how.md]: Good to know. Will delete.

**Pull request merged**